src/pools/abstract-pool.ts

```diff
@@ -199,6 +198,20 @@ export abstract class AbstractPool<
     }
   }
 
+  /**
+   * Increase the number of tasks that the given workers has done.
+   *
+   * @param worker Workers whose tasks are increased.
+   */
+  protected decreaseWorkersTasks (worker: Worker): void {
+    const numberOfTasksTheWorkerHas = this.tasks.get(worker)
```

Comment:
We could rename this local variable to `numberOfCurrentTasks` or similar

Reply:
Or as in dynamic cluster => `tasksInProgress`

Reply:
That is a copy and paste from the increased method, anyway I agree and will change on both